docs/modules/getting-started/pages/blue-green.adoc

```diff
@@ -0,0 +1,261 @@
+----
+
+Now, you have two separate Hazelcast Enterprise clusters running on your local.
+
+== Step 3. Configure the Client
+
+In this step, you'll configure a client, so that it initially connects to the `blue` cluster, and when
+the `blue` cluster is down, it automatically connects to the `green` cluster.
+
+For this, you need to create two client configurations for the same client, and pass these to a failover configuration.
+
+. Create the client configuration file for the `blue` cluster, called `client-blue.yaml` (or `client-blue.xml`).
++
+[tabs] 
+==== 
+YAML:: 
++ 
+-- 
+[source,shell]
+----
+hazelcast-client:
+  cluster-name: blue
+  network:
+    cluster-members:
+      - 172.18.0.2
+----
+--
+
+XML::
++
+[source,shell]
+----
+<hazelcast-client>
+    <cluster-name>blue</cluster-name>
+    <network>
+        <cluster-members>
+            <address>172.18.0.2</address>
+        </cluster-members>
+    </network>
+</hazelcast-client>
+----
+====
+. Create the client configuration for the `green` cluster, called `client-green.yaml` (or `client-green.xml`).
++
+[tabs] 
+==== 
+YAML:: 
++ 
+-- 
+[source,shell]
+----
+hazelcast-client:
+  cluster-name: green
+  network:
+    cluster-members:
+      - 172.18.0.3
+----
+--
+
+XML::
++
+[source,shell]
+----
+<hazelcast-client>
+    <cluster-name>green</cluster-name>
+    <network>
+        <cluster-members>
+            <address>172.18.0.3</address>
+        </cluster-members>
+    </network>
+</hazelcast-client>
+----
+====
+. Create the client failover configuration by passing the above two client configurations.
+Name of this configuration file must be `hazelcast-client-failover.yaml` (or `hazelcast-client-failover.xml`).
++
+[tabs] 
+==== 
+YAML:: 
++ 
+-- 
+[source,shell]
+----
+hazelcast-client-failover:
+  try-count: 4
+  clients:
+    - client-blue.yaml
+    - client-green.yaml
+----
+--
+
+XML::
++
+[source,shell]
+----
+<hazelcast-client-failover>
+    <try-count>4</try-count>
+    <clients>
+        <client>client-blue.xml</client>
+        <client>client-green.xml</client>
+    </clients>
+</hazelcast-client-failover>
+----
+====
++
+In this failover configuration file, you are directing the client to connect to the clusters in the given order from top to bottom;
+see xref:clients:java#ordering-of-clusters-when-clients-try-to-connect[Ordering of Clusters]. So, when you start the client
+(see Step 4 below), it will initially connect to the blue cluster. Here is what may happen:
+
+* When the blue cluster is gone, the client attempts to reconnect to it four times.
+* If not successful, it will try to connect to the green cluster, again, four times.
+* If these eight attempts are not successful, the client is shutdown.
+
+== Step 4. Connect the Client to Blue Cluster
+
+In this step, you'll start the client.
+
+[source,java]
+----
+HazelcastInstance client = HazelcastClient.newHazelcastFailoverClient();
```

Comment:
Why does this example only show an example with the Java client? The security example shows more client languages. Also, I feel like we are missing some context here. How does the client know how to find the config files we created?